### ISSUE TITLE
Error: Cannot add a Service with the same UUID '<UUID>' and subtype 'WLED-P  RESET-1' as another Service in this Accessory.

### ISSUE THREAD

**Issue by M0HID** [smhex/homebridge-wled-ws#99]
**Describe Your Problem:**
I keep getting this error, even after many attempts at resinstalling the plugin, and restarting the system
**Logs:**

```
Error: Cannot add a Service with the same UUID '00000049-0000-1000-8000-0026BB765291' and subtype 'WLED-P  RESET-1' as another Service in this Accessory.
    at Accessory.addService (/var/lib/homebridge/node_modules/homebridge/node_modules/hap-nodejs/src/lib/Accessory.ts:561:17)
    at PlatformAccessory.addService (/var/lib/homebridge/node_modules/homebridge/src/platformAccessory.ts:99:41)
    at WledWsPlatformAccessory.onPresetsReceived (/var/lib/homebridge/node_modules/homebridge-wled-ws/src/WledWsAccessory.ts:538:48)
    at /var/lib/homebridge/node_modules/homebridge-wled-ws/src/WledWsAccessory.ts:350:12
    at _class2.<anonymous> (/var/lib/homebridge/node_modules/homebridge-wled-ws/node_modules/wled-client/src/utils.emitter.ts:17:37)
    at _class2.[nodejs.internal.kHybridDispatch] (node:internal/event_target:786:20)
    at _class2.dispatchEvent (node:internal/event_target:721:26)
    at _class2.emit (/var/lib/homebridge/node_modules/homebridge-wled-ws/node_modules/wled-client/src/utils.emitter.ts:34:15)
    at _class2.setContext (/var/lib/homebridge/node_modules/homebridge-wled-ws/node_modules/wled-client/src/client.ts:190:21)
    at _class2.refreshContext (/var/lib/homebridge/node_modules/homebridge-wled-ws/node_modules/wled-client/src/client.ts:125:8)
```

**Plugin Config:**

```json
{
    "bridge": {
        "name": "Homebridge 84D4",
        "username": "0E:A7:B6:1E:77:51",
        "port": 51351,
        "pin": "846-66-665",
        "advertiser": "avahi"
    },
    "accessories": [],
    "platforms": [
        {
            "name": "Config",
            "port": 8581,
            "platform": "config"
        },
        {
            "name": "wled-ws platform",
            "controllers": [
                {
                    "name": "Light Strip",
                    "address": "wled-room.local",
                    "presets": "Hult,Rainbow,Rainbow-Split,Saw-Blue,Scan",
                    "showRealTimeModeButton": false,
                    "resetRealTimeModeAfterStream": true
                }
            ],
            "logging": true,
            "platform": "wled-ws"
        }
    ]
}
```

**Screenshots:**
![image](https://github.com/smhex/homebridge-wled-ws/assets/91326009/3e8fb98e-5a0e-4b31-890d-2d71b4098c25)
![image](https://github.com/smhex/homebridge-wled-ws/assets/91326009/4ba65cda-4885-490d-8612-f548f0dfaaec)

**Environment:**

* **Plugin Version**: v0.0.14
* **Homebridge Version**: v1.8.3
* **Operating System**: Raspberry Pi OS

**Comment by smhex** [comment]
Hi @M0HID ,

this looks like a bug being introduced with the latest release. I'll create a new release with a fix for it.

Thanks for reporting it!

**Comment by smhex** [comment]
Hi @M0HID ,

I created a new version [0.0.15-beta.0](https://www.npmjs.com/package/homebridge-wled-ws/v/0.0.15-beta.0). Can you please test and report your feedback here? 

I recommend to remove your WLED controller before upgrading using "Homebridge->Settings->Remove device from cache". This will avoid orphaned services being a result of this bug. Normally the plugin takes care of adding/removing services.

Thank you!

**Comment by smhex** [comment]
Fixed in 0.0.15+
